app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,181 @@
+<form class="ui form" autocomplete="off" {{action 'submit' on='submit' }}>
```

Comment:
The test is failing due to `we're` in 
"After 10:00 minutes, the reservation we're holding will be released to others."